src/dragonball/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019-2022 Alibaba Cloud. All rights reserved.
+# Copyright (c) 2019-2022 Ant Group. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+default: build
+
+build:
+    cargo update -p vm-memory:0.8.0 --precise 0.7.0
```

Comment:
This is because the crates in dbs-xxx uses version 0.7.0 of vm-memory, but rust-vmm uses 0.8.0. This will be resolved by upgrading dbs-xxx in a later patch.

Reply:
space is changed to tab